bundle-patch/patch_csv.yaml

```diff
@@ -134,6 +134,7 @@ spec:
     ### Support & Troubleshooting
 
     Red Hat build of OpenTelemetry is available as part of a Red Hat OpenShift subscription.
+    Checking the [Red Hat Documentation](https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/red_hat_build_of_opentelemetry/index) is recommended when installing, configuring, and managing the Operator and instances.
```

Comment:
Is there a way to avoid `4.17` in the URL and point the link always to the latest version?

Reply:
Unfortunately, as of now, it is not possible. I hope that we will be able to use `latest` when this issue is resolved.